runtime/type_test.go

```diff
@@ -548,3 +548,12 @@ func newTestClassStrictEq(name string, base *Type) *Type {
 	}))
 	return t
 }
+
+func newTestIndexObject(index int) *Object {
```

Comment:
Not sure this belongs in type_test.go. Maybe just put it in builtin_types_test.go for now.